packages/db/src/sql/sqlite.ts

```
@@ -1,286 +1,242 @@
+        createTables(this, tables, indexes, initializeValues, logger, resolve, reject);
       } else {
-        import('sqlite3')
+        import("sqlite3")
           .then((sqlite3) => {
             // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
-            // @ts-ignore
-            if (sqlite3.Database == null) sqlite3 = sqlite3.default
-            const db = (this.db = new sqlite3.Database(conf.database_host))
+            // @ts-expect-error
+            if (!sqlite3.Database) sqlite3 = sqlite3.default;
+            const db = (this.db = new sqlite3.Database(conf.database_host));
```

Biome (2.4.9)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)